config.py.sample

```diff
@@ -57,3 +57,6 @@ MEMCACHED_NAMESPACE = "AB"
 #MAIL_FROM_DOMAIN = "acousticbrainz.org"
 
 FILE_STORAGE_DIR = "./files"
+
+# Feature Flags
+FEATURE_EVAL_LOCATION = False
```

Comment:
I'm wondering if we should add a file with default configuration values so that people who use the server don't have to copy stuff from example one all the time unless it's something critical. See https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/default_config.py.